x/sp/client/cli/query.go

```diff
@@ -93,3 +99,41 @@ func CmdStorageProvider() *cobra.Command {
 
 	return cmd
 }
+
+func CmdStorageProviderByOperatorAddress() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "storage-provider-by-operator-address [operator address]",
+		Short: "Query StorageProviderByOperatorAddress",
```

Comment:
refine the short `"Query StorageProviderByOperatorAddress"`

Reply:
FIxed